configure.py

```diff
@@ -482,6 +550,24 @@ def _boringssl_setup(cfg):
             ),
         ]
         return cmds
+    elif cfg.host == "iOS":
+        build_type = "Debug" if cfg.args.buildtype == "debug" else "Release"
+        cmds = [
+            _cmd_join(
+                "cmake",
+                "-GNinja",
+                f"-DCMAKE_BUILD_TYPE={build_type}",
+                f"-DCMAKE_INSTALL_PREFIX={cfg.prefix}",
+                f"-DCMAKE_OSX_SYSROOT=iphoneos",
+                f"-DCMAKE_OSX_ARCHITECTURES={cfg.ios_abi}",
+                "-DBUILD_SHARED_LIBS=OFF",
+                "-S",
+                cfg.externals["boringssl"],
+                "-B",
+                _get_builddir(cfg, "boringssl"),
```

Comment:
can you check if there isn't an SSL builtin on iOS?

Reply:
Done by using `--enable-securetransport`.

Reply:
Nevermind, while it compiled, the API used by this tls backend are deprecated by Apple. Thus, I re-enabled the build  of the boringssl lib.